homeassistant/components/yale_smart_alarm/config_flow.py

```diff
@@ -66,6 +91,14 @@ async def async_step_reauth_confirm(self, user_input=None):
                     data_schema=DATA_SCHEMA,
                     errors={"base": "invalid_auth"},
                 )
+            except requests.HTTPError as error:
+                if "401 Client Error" in str(error):
```

Comment:
My impression is that you can check `error.response.status_code == 401`  or something like that here.

Reply:
Corrected in commit